### DISCUSSION TITLE
关于推理加速

### DISCUSSION THREAD

**Discussion by 1042280163** [fishaudio/fish-speech#711] [General]
想知道这个compile加速，这个加速能适配到别的语音模型吗，像cosyvoice，f5-tts这些。还有就是这个模型总共多少参数量啊

**Comment by None1145** [comment]
compile加速应该是可以适配到其他模型上的(没仔细看代码), 参数量不清楚

**Comment by Stardust-minus** [comment]
Plz use English to create discussion, tks :)
